Header header

float32 speed                     # m/s